packages/devtools_app/lib/src/profiler/cpu_profile_transformer.dart

```diff
@@ -46,6 +46,23 @@ class CpuProfileTransformer {
     _stackFrameKeys = cpuProfileData?.stackFramesJson?.keys?.toList() ?? [];
     _stackFrameValues = cpuProfileData?.stackFramesJson?.values?.toList() ?? [];
 
+    // Initialize all stack frames before we start to for the tree.
+    for (int i = 0; i < _stackFramesCount; i++) {
+      final k = _stackFrameKeys[i];
```

Comment:
Uber nit: I'm personally not a fan of single character variables for anything other than iterators. Maybe change these to `key` and `values`?

Reply:
done